etc/docs/template/layouts/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    {{ partial "meta.html"}}
+    <link rel="shortcut icon" href="{{.Site.BaseURL}}/s/img/favicon.png">
```

Comment:
The icon should be updated: https://www.mongodb.com/assets/images/global/favicon.ico

Reply:
same comment as other `img` changes